src/Glpi/Application/ErrorHandler.php

```diff
@@ -259,20 +264,20 @@ public function register(): void
      */
     public function handleError(int $error_code, string $error_message, string $filename, int $line_number)
     {
-
        // Have to false to forward to PHP internal error handler.
         $return = !$this->forward_to_internal_handler;
 
         $trace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
         array_shift($trace);  // Remove current method from trace
         $error_trace = $this->getTraceAsString($trace);
 
-        if (in_array($error_code, self::FATAL_ERRORS)) {
-           // Fatal errors are handled by shutdown function
-           // (as some are not recoverable and cannot be handled here).
-           // Store backtrace to be able to use it there.
+        if (in_array($error_code, self::FATAL_ERRORS, true)) {
+            // Fatal errors are handled by shutdown function
+            // (as some are not recoverable and cannot be handled here).
+            // Store backtrace to be able to use it there.
             $this->last_fatal_trace = $error_trace;
```

Comment:
As fatal erros are now converted to exceptions, I guess there is no need to store this trace anymore, as fatal errors are now converted to exceptions.
```suggestion
```